test/suites/drift/suite_test.go

```diff
@@ -279,7 +279,10 @@ var _ = Describe("Drift", Label("AWS"), func() {
 				SecurityGroupSelector: map[string]string{"karpenter.sh/discovery": settings.FromContext(env.Context).ClusterName},
```

Comment:
This doesn't have a `go.mod` bump so this test is going to fail if it gets merged, right?

Reply:
Yeah, just waiting to merge the core PR